### PR TITLE
added new underground fluids and values

### PR DESCRIPTION
**Pull request by Dream-Master**
added new underground fluids and values
first suggestions please discuss and suggest new values
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5974

### CODE REVIEW

**Comment by GTNH-Colen** [comment]
Rundown of the above changes:

Screenshot 2020-04-25 at 19 14 23

**Comment by quedral** [comment]
Nice addition in my opinion.

File: config/GregTech/GregTech.cfg [hunk 1]
```diff
@@ -299,6 +299,23 @@ undergroundfluid {
...
 
+        molten_iron {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=30
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=800
```
Comment (bartimaeusnek) [inline config/GregTech/GregTech.cfg:310]: 800 is quite high for a metal, id go with a lower value

File: config/GregTech/GregTech.cfg [hunk 2]
```diff
@@ -322,6 +339,40 @@ undergroundfluid {
...
 
+        sulfuric_acid {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=100
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=500
```
Comment (bartimaeusnek) [inline config/GregTech/GregTech.cfg:350]: maybe cut this in half aswell...

File: config/GregTech/GregTech.cfg [hunk 3]
```diff
@@ -364,6 +415,63 @@ undergroundfluid {
...
+
+        phosphoric_acid {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=80
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=800
```
Comment (bartimaeusnek) [inline config/GregTech/GregTech.cfg:464]: 800 is too high for a pure substance, that has direct use

File: config/GregTech/GregTech.cfg [hunk 4]
```diff
@@ -484,6 +609,52 @@ undergroundfluid {
...
+
+        hydrogen_sulfide {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=100
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=600
```
Comment (bartimaeusnek) [inline config/GregTech/GregTech.cfg:624]: this could be increased, since it needs processing, maybe 800-1000

File: config/GregTech/GregTech.cfg [hunk 5]
```diff
@@ -579,6 +773,69 @@ undergroundfluid {
...
+
+        hydrofluoric_acid {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=80
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=600
```
Comment (bartimaeusnek) [inline config/GregTech/GregTech.cfg:807]: 300 at max, rather 200-250

**Comment by 0lafe** [comment]
Bruh you still gotta removal Nat gas. I'm not sure how people don't understand this after 2 people explained it to them, but realism/ignorance doesn't make it OP. 

Oil gives Nat gas. Pumping Nat gas is like pumping sulfuric LF. The main bottleneck in any oil based production is gonna be distillation, which natgas entirely bypasses. In a 2 step process it'll beat almost all fuels, and with a 3 step production of methane, it'll do better than anything I've seen for the setup.

**Comment by GTNH-Colen** [comment]
> Bruh you still gotta remove Nat gas. I'm not sure how people don't understand this after 2 people explained it to them, but realism/ignorance doesn't make it not OP.

Why don't you make a real RFC thread for it and discuss it there rather than hiding it in a pull request to try force a stealth nerf like you did earlier.

**Comment by 0lafe** [comment]
Lmao

File: config/GregTech/GregTech.cfg (hunk 1 above)
Comment (Dream-Master) [reply config/GregTech/GregTech.cfg:310]: changed to 400

File: config/GregTech/GregTech.cfg (hunk 2 above)
Comment (Dream-Master) [reply config/GregTech/GregTech.cfg:350]: changed to 250

File: config/GregTech/GregTech.cfg (hunk 3 above)
Comment (Dream-Master) [reply config/GregTech/GregTech.cfg:464]: changed to 400

Comment (Dream-Master) [reply config/GregTech/GregTech.cfg:464]: need to be tested why it not worked in the first place

File: config/GregTech/GregTech.cfg (hunk 4 above)
Comment (Dream-Master) [reply config/GregTech/GregTech.cfg:624]: change to 900

File: config/GregTech/GregTech.cfg (hunk 5 above)
Comment (Dream-Master) [reply config/GregTech/GregTech.cfg:807]: chane to 300

**Comment by Dream-Master** [comment]
@bartimaeusnek so i cahnge al values please recheck. Thanks

**Pull request merged**